packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -58,7 +62,11 @@ export const reduceCallControlsForMobile = (
 enum CallEndReasons {
   LEFT_CALL,
   ACCESS_DENIED,
-  REMOVED_FROM_CALL
+  REMOVED_FROM_CALL,
+  /* @conditional-compile-remove(rooms) */
```

Comment:
This enum is not exported. Remove conditional compilation directives.

This may allow you to remove conditional compilation from logic that uses this enum as well.